CWE/events/italy.txt

```diff
@@ -54,460 +99,232 @@ country_event = {
+		clr_country_flag = semi_presidential_system_republic
+		clr_country_flag = parliamentary_system_republic
+			
+		set_country_flag = parliamentary_system_republic
 
-	option = {
-		name = EVT_8007735_B
-		ai_chance = { factor = 0.2 }
-		any_pop = { militancy = 3 }
-		any_pop = {
-			limit = { is_primary_culture = yes }
-			ideology = { factor = 0.05 value = conservative }
-		}
-		set_country_flag = very_radical_ICP 
+		ai_chance = { factor = 1 }
 	}
 }
 
-#Togliatti assassination attempt
+#1948 elections
 country_event = {
-  id = 8007736
-  title = EVT_8007736_NAME
-  desc = EVT_8007736_DESC
-  picture = "nwo2_palmiro_togliatti"
-  fire_only_once = yes
-
-	trigger = {
-		tag = ITA
-		year = 1948 NOT = { year = 1949 }
-		OR = { government = democracy government = hms_government }
-		has_country_flag  = very_moderate_ICP
-		NOT = { 
-			OR = { 
-				ruling_party_ideology = socialist 
-				ruling_party_ideology = communist 
-				ruling_party_ideology = communist_social 
-			} 
-		}
-	}
-
-	mean_time_to_happen = { months = 2 }
-
-	option = {
-		name = EVT_8007736_A
-		any_pop = { consciousness = 10 }
-		ai_chance = { factor = 0.6 }
-	}
-
-	option = {
-		name = EVT_8007736_B
-		any_pop = { consciousness = 5 militancy = 5 }
-		any_pop = {
-			limit = { is_primary_culture = yes }
-			ideology = { factor = 0.025 value = communist }
-		}
-		set_country_flag = radical_ICP
-		ai_chance = { factor = 0.2 }
-	}
-
-	option = {
-		name = EVT_8007736_C
-		any_pop = { militancy = 10 }
-		any_pop = {
-			limit = { is_primary_culture = yes }
-			ideology = { factor = -0.05 value = communist }
-		}
-		set_country_flag = very_radical_ICP
-		ai_chance = { factor = 0.2 }
-	}
+	id = 8007771
+	title = "$YEAR$ Italian General Election"
+	desc = EVTDESC8007771
+	picture = "ITA_1948_elections"
 	
-}
-
-#Italian Communist Party in 1950s
-country_event = {
-  id = 8007738
-  title = EVT_8007738_NAME
-  desc = EVT_8007738_DESC
-  picture = "nwo2_italian_communist_party"
-  fire_only_once = yes
+	fire_only_once = yes
 
 	trigger = {
 		tag = ITA
-		year = 1950 NOT = { year = 1952 }
-		has_country_flag  = very_moderate_ICP
-		OR = { government = democracy government = hms_government }
-		NOT = { 
-			OR = { 
-				ruling_party_ideology = socialist 
-				ruling_party_ideology = communist 
-				ruling_party_ideology = communist_social 
-			} 
-		}
-		has_global_flag = blocsenabled
+		year = 1948
+		NOT = { year = 1950 }
+		
+		government = democracy
+		war = no
+		is_vassal = no
+	
 	}
 
-	mean_time_to_happen = { months = 2 }
+	mean_time_to_happen = { months = 4 }
 
 	option = {
-		name = EVT_8007738_A
-		any_pop = { consciousness = 5 }
-		any_pop = {
-			limit = { is_primary_culture = yes }
-			ideology = { factor = 0.05 value = socialist }
+		name = "Centrist Coalition (DC-US-BN-PRI)"
+		ruling_party_ideology = conservative
+		any_pop = { 
+			ideology = { factor = 0.1 value = conservative }
 		}
+		set_country_flag = 1948_ITA_Centrism
 		ai_chance = { factor = 0.7 }
```

Comment:
If there are 2 options in a historical event, the minimum chance for the historical option must be 90%.

Reply:
wait so no 70/30 possibilities if it was a tight race OTL?

Reply:
Yeah, I like it at 90% so it’s easier for testing and also so people don’t keep complaining that ahistorical events keep happening and spoiling their game.